pkg/services/ngalert/notifier/alertmanager.go

```diff
@@ -49,6 +61,50 @@ func (am *Alertmanager) IsDisabled() bool {
 
 func (am *Alertmanager) Init() error {
 	am.logger = log.New("alertmanager")
+	am.Store = store.DBstore{} //TODO: Is this right?
```

Comment:
I thought I didn't need to do that. NG does it because it was to re-use the injected SQL store throughout other components.

I thought:
```
	SQLStore        *sqlstore.SQLStore       `inject:""`
````

In the `Alertmanager` struct would take care of that, am I wrong?